static/css/devhub/new-landing/base.less

```diff
@@ -23,7 +23,7 @@
 
 // Light
 @color-scheme-light-text: #333;
-@color-scheme-light-background: #f1f3f6;
+@color-scheme-light-background: #f3f5f8;
```

Comment:
The color apparently changed in one of the older sketch files. This is now the correct one.